packages/drafts-realm/chess.gts

```diff
@@ -0,0 +1,466 @@
+import {
+  Component as BoxelComponent,
+  CardDef,
+} from 'https://cardstack.com/base/card-api';
+import Component from '@glimmer/component';
+import StringCard from 'https://cardstack.com/base/string';
+import { contains, field } from 'https://cardstack.com/base/card-api';
+
+import lodash from 'https://cdn.jsdelivr.net/npm/lodash@4.17.21/+esm';
+import {
+  Chessboard,
+  FEN,
+} from 'https://cdn.jsdelivr.net/npm/cm-chessboard@8.7.3/+esm';
```

Comment:
I dono whether it was unpkg or the library itself but I had issues using unpkg with chessboard.js. But, that had other dependencies like jquery and more stylesheets. 

I had a bettere experience using jsdelivr -- my 1st step was actually just getting lodash import to work just to test a utility library

Reply:
I think the new way to import is using esm.run 

'https://esm.run/cm-chessboard@8.7.3' 

This would return the esm build rather than loading js files